cinema/models.py

```diff
@@ -38,16 +44,24 @@ def full_name(self):
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
-    duration = models.IntegerField()
+    duration = models.IntegerField(default=90)
     genres = models.ManyToManyField(Genre)
     actors = models.ManyToManyField(Actor)
+    image = models.ImageField(upload_to="uploads/movies/", blank=True)
```

Comment:
The `image` field in the `Movie` model is set to `blank=True` but not `null=True`. This means that the database will require a value for this field, even if it's an empty string. If you want to allow the field to be completely optional, consider adding `null=True`.